### PR TITLE
Sealed classes

### PR DESCRIPTION
**Pull request by ianardee**
* :recycle: :boom: seal product classes
*  :recycle: declare lists as interfaces
* :bug: fix page output in us bank check
* :sparkles: add us driver license
* :white_check_mark: add tests for empty responses

### CODE REVIEW

File: src/Mindee/Product/Us/BankCheck/BankCheckV1.cs [hunk 1]
```diff
@@ -4,10 +4,10 @@
 namespace Mindee.Product.Us.BankCheck
 {
     /// <summary>
-    /// The bank check v4 definition.
+    /// The definition for bank_check v1.
```
Comment (sebastianMindee) [inline src/Mindee/Product/Us/BankCheck/BankCheckV1.cs:7]:
```suggestion
    /// The definition for Bank Check, API version 1.
```

**Review by sebastianMindee — CHANGES_REQUESTED** [review]
Wrong name for latest inference descriptions

File: src/Mindee/Product/Us/DriverLicense/DriverLicenseV1.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+using Mindee.Http;
+using Mindee.Parsing.Common;
+
+namespace Mindee.Product.Us.DriverLicense
+{
+    /// <summary>
+    /// The definition for us_driver_license v1.
```
[the remark this answers is not in the document]
Comment (ianardee) [reply src/Mindee/Product/Us/DriverLicense/DriverLicenseV1.cs]: fixed, thanks

**Review by sebastianMindee — APPROVED** [review]
